<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgFog</class>
 <widget class="QDialog" name="CQDlgFog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fog Settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QCheckBox" name="qqFogEnabled">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Fog enabled</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Fog type</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="qqLinear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Linear</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqExp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>25</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Exp</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqExp2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>25</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Exp2</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Start distance [m]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqStartDistance">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>76</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>End distance [m]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqEndDistance">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>76</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqDensity">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>76</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Density</string>
   </property>
  </widget>
  <widget class="QToolButton" name="qqAdjustColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>180</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Adjust fog color</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqDensity_2">
   <property name="geometry">
    <rect>
     <x>2520</x>
     <y>410</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqStartDistance_2">
   <property name="geometry">
    <rect>
     <x>2520</x>
     <y>350</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqEndDistance_2">
   <property name="geometry">
    <rect>
     <x>2520</x>
     <y>380</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqFogEnabled</tabstop>
  <tabstop>qqLinear</tabstop>
  <tabstop>qqExp</tabstop>
  <tabstop>qqExp2</tabstop>
  <tabstop>qqStartDistance</tabstop>
  <tabstop>qqEndDistance</tabstop>
  <tabstop>qqDensity</tabstop>
  <tabstop>qqAdjustColor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
